docs/src/reference/lib/changelog.rst

```diff
@@ -10,6 +10,11 @@ Added
 * Add thread limiting context manager utility (See `KEP 2: Thread Management <https://docs.klio.io/en/latest/keps/kep-002.html>`_).
 * Add default thread management to ``@handle_klio`` decorator (See `KEP 2: Thread Management <https://docs.klio.io/en/latest/keps/kep-002.html>`_).
 
+Fixed
+*****
+
+* Partially reverted back to reading config from effective-klio-job.yaml.
```

Comment:
Non-blocking but might be nice to link the PR # that the Runtime config was introduced. And potentially adding a little but detail on the error users were seeing leading to this partial revert!